src/network/clientpackethandler.cpp

```diff
@@ -219,6 +219,16 @@ void Client::handleCommand_AccessDenied(NetworkPacket* pkt)
 		u8 reconnect;
 		*pkt >> reconnect;
 		m_access_denied_reconnect = reconnect & 1;
+	} else if (denyCode == SERVER_ACCESSDENIED_TRANSFER) {
+		std::string addressPack;
+		*pkt >> addressPack;
+
+		m_access_denied_reason = accessDeniedStrings[denyCode];
```

Comment:
Being network-facing code, I think this should be more careful. Shouldn't there be an index check here?

Reply:
I think that this is fine because of the condition above. `denyCode` has to be equal to `SERVER_ACCESSDENIED_TRANSFER` here.

Reply:
Yep, that's true. (Though I don't see why we do the array lookup then - might as well hardcode the string here...)